scripts/run_pylint.py

```diff
@@ -56,7 +56,7 @@
     },
     'BASIC': {
         'method-rgx': '[a-z_][a-z0-9_]{2,40}$',
-        'function-rgx': '[a-z_][a-z0-9_]{2,40}$',
+        'function-rgx': '[a-z_][a-z0-9_]{2,45}$',
```

Comment:
Sadness. Why not disable it locally? Churn in your lint RC on a run-of-the-mill PR is a bad sign.